packages/pigment-css-react/README.md

```diff
@@ -39,7 +39,9 @@ Pigment CSS is built on top of [WyW-in-JS](https://wyw-in-js.dev/), enabling to
 
 ### Start with Next.js
 
-Use the following commands to create a new Next.js project with Pigment CSS set up:
+#### Quickstart
```

Comment:
Felt like a title like this one was missing in opposition to the "Manual installation". I know it can sound a bit dumb, but I wondered about this for a couple of seconds before starting 😅 The extra clarity could be worth it.